src/main/java/com/forest/mytopmovies/utils/ExternalMovieDBApiUtil.java

```diff
@@ -35,12 +39,24 @@
         });
         List<Movie> convertedMovies = PojoEntityParamDtoConverter.convertMoviePojoListToEntityList(queryResult.getResults(), genreService);
 
+        // store to cache
+        convertedMovies.forEach(movie -> {
+            LOGGER.debug("Saving movie with id {} to cache.", movie.getTmdbId());
+            cacheManager.getCache(cacheName).put(movie.getTmdbId(), movie);
```

Comment:
cacheManager is usually wrapped in Objects.requireNonNull, alt-enter on intellij will usually do this automatically for you.